articles/windows-linux-setup-experience.md

```diff
@@ -0,0 +1,45 @@
+# Installing software during setup on Windows and Linux
+
+_Available in Fleet Premium_
+
+In Fleet, you can customize the software that's installed while enrolling new users on Windows and Linux, through a process called Setup Experience.
+
+This feature is similar to the setup experience [available on macOS](https://fleetdm.com/guides/macos-setup-experience) during ADE enrollment.
```

Comment:
Setup experience shouldn't be capitalized (we don't treat feature names as proper nouns), but for some reason the wording feels odd when it's all lowercased. What do you think of this: 
```suggestion
In Fleet, you can customize the software that's installed while enrolling new users on Linux. This feature is similar to the setup experience [available on macOS](https://fleetdm.com/guides/macos-setup-experience) during ADE enrollment.
```